core/src/test/scala/org/apache/spark/sql/IssueTestSuite.scala

```diff
@@ -19,6 +19,29 @@ import com.pingcap.tispark.TiConfigConst
 import org.apache.spark.sql.functions.{col, sum}
 
 class IssueTestSuite extends BaseTiSparkSuite {
+  test("adding set and enum") {
+    tidbStmt.execute("drop table if exists set_t")
+    tidbStmt.execute("drop table if exists enum_t")
+    tidbStmt.execute(
+      "CREATE TABLE `set_t` (" +
+        "`priority` set('Low','Medium','High') NOT NULL)"
```

Comment:
I think it would be better if you drop those tables explicitly in afterAll() method.